components/esp_eth/include/esp_eth_phy.h

```diff
@@ -304,6 +297,20 @@ esp_eth_phy_t *esp_eth_phy_new_ksz8041(const eth_phy_config_t *config);
 */
 esp_eth_phy_t *esp_eth_phy_new_ksz8081(const eth_phy_config_t *config);
 
+/**
+* @brief Create a PHY instance of KSZ8091
+*
+* @param[in] config: configuration of PHY
+*
+* @return
+*      - instance: create PHY instance successfully
+*      - NULL: create PHY instance failed because some error occurred
+*/
+static inline esp_eth_phy_t *esp_eth_phy_new_ksz8091(const eth_phy_config_t *config)
+{
+	return esp_eth_phy_new_ksz8081(config);
```

Comment:
It was found the `tab` is used instead of spaces during the internal review. Could you please fix it?

Reply:
Done.